vault/resource_ldap_secret_backend_static_role_test.go

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package vault
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-provider-vault/internal/consts"
+	"github.com/hashicorp/terraform-provider-vault/internal/provider"
+	"github.com/hashicorp/terraform-provider-vault/testutil"
+)
+
+func TestAccLDAPSecretBackendStaticRole(t *testing.T) {
+	var (
+		mount                 = acctest.RandomWithPrefix("tf-test-ldap-static-role")
+		bindDN, bindPass, url = testutil.GetTestLDAPCreds(t)
+		resourceType          = "vault_ldap_secret_backend_static_role"
+		resourceName          = resourceType + ".role"
+		username              = "alice"
+		dn                    = "cn=alice,ou=users,dc=example,dc=org"
+		rotationPeriod        = "60"
+		updatedUsername       = "bob"
+		updatedDN             = "cn=bob,ou=users,dc=example,dc=org"
+		updatedRotationPeriod = "120"
+	)
+	fmt.Println(testLDAPSecretBackendStaticRoleConfig(mount, bindDN, bindPass, url, username, dn, username, rotationPeriod))
```

Comment:
Is this Println left over from debugging?

Reply:
oops, removed 👍🏼